devtools/cli/src/main/java/io/quarkus/cli/RegistryAddCommand.java

```diff
@@ -0,0 +1,74 @@
+package io.quarkus.cli;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import io.quarkus.cli.registry.BaseRegistryCommand;
+import io.quarkus.cli.registry.RegistryClientMixin;
+import io.quarkus.registry.config.RegistriesConfig;
+import io.quarkus.registry.config.RegistriesConfigLocator;
+import io.quarkus.registry.config.RegistryConfig;
+import io.quarkus.registry.config.json.JsonRegistriesConfig;
+import io.quarkus.registry.config.json.JsonRegistryConfig;
+import io.quarkus.registry.config.json.RegistriesConfigMapperHelper;
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "add", sortOptions = false, showDefaultValues = true, mixinStandardHelpOptions = false, header = "Add a Quarkus extension registry to the registry client configuration", description = "%n"
+        + "This command will add a Quarkus extension registry to the registry client configuration unless it's already present", headerHeading = "%n", commandListHeading = "%nCommands:%n", synopsisHeading = "%nUsage: ", parameterListHeading = "%n", optionListHeading = "Options:%n")
+public class RegistryAddCommand extends BaseRegistryCommand {
+
+    @CommandLine.Mixin
+    protected RegistryClientMixin registryClient;
+
+    @CommandLine.Parameters(arity = "0..1", paramLabel = "REGISTRY-ID[,REGISTRY-ID]", description = "Registry ID to add to the registry client configuration")
```

Comment:
Add an example value to the description, e.g. https://github.com/quarkusio/quarkus/blob/fb575c117f6a4958bcf2a1f21a134fc7b98b1198/devtools/cli/src/main/java/io/quarkus/cli/create/TargetGAVGroup.java#L15